src/ui/pages/SidePage/components/IncomingRequest/components/SignTransactionRequest.tsx

```diff
@@ -20,38 +20,30 @@ const SignTransactionRequest = ({
   handleCancel,
 }: RequestProps) => {
   const signTransaction = requestData.signTransaction;
+  const [isSigningObject, setIsSigningObject] = useState(false);
   const ballotLogo = requestData.logo ? requestData.logo : CardanoLogo;
 
   const signDetails = useMemo(() => {
     if (!signTransaction) return {};
```

Comment:
annoying request but since this is just sign and not sign transaction can you update those variable names?